stamina-testkit/src/main/scala/stamina/testkit/StaminaTestKit.scala

```diff
@@ -0,0 +1,85 @@
+package stamina
+package testkit
+
+import scala.util._
+import scala.annotation.tailrec
+
+trait StaminaTestKit { self: org.scalatest.WordSpecLike ⇒
+
+  val defaultSampleId = "default"
+  case class PersistableSample(sampleId: String, persistable: AnyRef, description: Option[String]) {
+    override def toString = persistable.getClass.getSimpleName + description.map(" " + _).getOrElse("")
+  }
+
+  def sample(persistable: AnyRef) = new PersistableSample(defaultSampleId, persistable, None)
+  def sample(sampleId: String, persistable: AnyRef) = new PersistableSample(sampleId, persistable, Some(sampleId))
+  def sample(sampleId: String, persistable: AnyRef, description: String) = new PersistableSample(sampleId, persistable, Some(description))
+
+  implicit class TestablePersisters(persisters: Persisters) extends org.scalatest.Matchers {
```

Comment:
Hmm, getting confusing clashes when importing `org.scalatest.Matchers._`, feel free to have a crack at it if you want :)